### PR TITLE
Assorted pricing/user liaison work 

### PR DESCRIPTION
**Pull request by zuluecho9**
Assorted edits, including: 

Some work related to DOC-6831, adding more links to the 'cancel/delete account doc'. 
Adding caveat to a usage query.

### CODE REVIEW

File: src/content/docs/accounts/accounts-billing/account-setup/downgradecancel-account.mdx [hunk 1]
```diff
@@ -52,7 +52,9 @@ Options for downgrading your New Relic organization will differ depending on you
 
 ## Cancel organization [#cancel]
 
-Options for cancelling your New Relic organization depend on your pricing edition:
+If you've accidentally signed up for New Relic and want to delete that account, see [Delete simple organization](#cancel-simple-org).
```
Comment (urbiz-nr) [inline src/content/docs/accounts/accounts-billing/account-setup/downgradecancel-account.mdx:55]: your account?

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles.mdx [hunk 2]
```diff
@@ -101,15 +99,18 @@ Here are some example user management procedures:
...
   >
     To add or edit users, use the [User management](#where) UI.
 
-    To add users there:
+    To add users:
 
     1. If you have multiple [authentication domains](/docs/accounts/accounts-billing/new-relic-one-pricing-users/configure-authentication-domains), choose one from the authentication domain dropdown.
     2. Click **Add user**.
-    3. Complete the process, including choosing [user type](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) and [user group](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-group).
+    3. Complete the process, including choosing [user type](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-type) and [user group](/docs/accounts/accounts-billing/new-relic-one-pricing-users/users-roles#user-group).
```
Comment (urbiz-nr) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles.mdx:110]: choosing _the_ user type?

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users.mdx [hunk 3]
```diff
@@ -16,11 +16,17 @@ watermark: |-
...
+## Before you start [#before-start]
 
-We provide several ways for you to manage your New Relic users. Here are our main docs related to managing users:
+Some important things to note before starting: 
+
+* This doc and the docs in this section are for managing users on the New Relic One user model. [Learn about user models.](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model#user-models)
+* For managing users on our original user model, see [Original users](/docs/accounts/original-accounts-billing/original-users-roles). 
+
+
+## User management docs [#docs]
```
Comment (urbiz-nr) [inline src/content/docs/accounts/accounts-billing/new-relic-one-user-management/introduction-managing-users.mdx:27]: I'd use a longer anchor link.

File: src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx [hunk 4]
```diff
@@ -77,11 +77,11 @@ Your data ingest is one factor in [New Relic One pricing](/docs/accounts/account
...
 * If you want to see a view over a long time, go to the time picker on the right side of the page, where it says **Since 30 days ago** and select a different time range.
 * To see more detail about ingest, scroll down on the data ingestion UI page to see an ingest breakdown.
-* We currently don't offer the ability to drill down into every type of data ingest. To do more in-depth analysis, use NRQL queries. Here's a query example of breaking down [`Metric` data](/docs/telemetry-data-platform/understand-data/new-relic-data-types/#dimensional-metrics) by top 10 metric names. To adjust your analysis, you might facet by `appName` or `Host`.
+* We currently don't offer the ability to drill down into every type of data ingest. To do more in-depth analysis, use NRQL queries. Here's a query example of breaking down [`Metric` data](/docs/telemetry-data-platform/understand-data/new-relic-data-types/#dimensional-metrics) by top 10 metric names. Note that some metrics are not data ingest metrics and are not billable. To run different analyses, you might also try faceting by `appName` or `Host`.
```
Comment (urbiz-nr) [inline src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx:80]: aren't

**Review by urbiz-nr — APPROVED** [review]
Looks good, left a few comments.

File: src/content/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles.mdx (hunk 2 above)
Comment (zuluecho9) [reply src/content/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles.mdx:110]: think that could go either way, think it's fine

**Pull request merged**